src/tests/test_data/dummy_storage.py

```diff
@@ -0,0 +1,14 @@
+from abc import ABC
+from typing import Any
+
+from qilib.utils.storage import StorageMongoDb
+
+
+class DummyStorage(StorageMongoDb, ABC):
+
+    def __init__(self, name, **kwargs):
+        """ Dummy storage used for testing."""
+        super().__init__(name, **kwargs)
+
+    def encode_serialize_data(self, data: Any) -> Any:
+        return super()._encode_data(self._serialize(data))
```

Comment:
why super()._encode_data instead of self._encode_data? ABC doesn't have this method for sure

Reply:
Replaced  super with self